presto-main/src/main/java/io/prestosql/sql/planner/plan/TableWriterNode.java

```diff
@@ -451,4 +452,56 @@ public String toString()
             return handle.toString();
         }
     }
+
+    public static class UpdateTarget
+            extends WriterTarget
+    {
+        private final TableHandle handle;
+        private final SchemaTableName schemaTableName;
+        private final List<String> updatedColumns;
+        private final List<ColumnHandle> updatedColumnHandles;
+
+        @JsonCreator
+        public UpdateTarget(
+                @JsonProperty("handle") TableHandle handle,
+                @JsonProperty("schemaTableName") SchemaTableName schemaTableName,
+                @JsonProperty("updatedColumns") List<String> updatedColumns,
+                @JsonProperty("updatedColumnHandles") List<ColumnHandle> updatedColumnHandles)
+        {
+            this.handle = requireNonNull(handle, "handle is null");
+            this.schemaTableName = requireNonNull(schemaTableName, "schemaTableName is null");
+            this.updatedColumns = requireNonNull(updatedColumns, "updatedColumns is null");
+            this.updatedColumnHandles = requireNonNull(updatedColumnHandles, "updatedColumnHandles is null");
```

Comment:
Verify that updateColumns and updatedColumnHandles have the same length.

Reply:
Added.